### PR TITLE
Chat2

### PR DESCRIPTION
**Pull request by ggpepp**
It finally works correctly. Login in time is extended because it has to also connect to the chat before the login can complete but it is worth it.

### CODE REVIEW

File: lib/views/mentor/mentorscreen.dart [hunk 1]
```diff
@@ -36,7 +30,7 @@ class _MentorScreenState extends State<MentorScreen> {
         children: [
           MentorHome(),
           Text("Second Screen for Mentor", textScaleFactor: 2),
-          Text("Third Screen for Mentor", textScaleFactor: 2),
+          MentorChatScreen(),
```
Comment (chwerth) [inline lib/views/mentor/mentorscreen.dart:33]: Could you move this here: https://github.com/PrepHQ/PrepHQ_Connect/blob/master/lib/views/student/mentorprofilescreen.dart#L71

**Review by chwerth — DISMISSED** [review]

File: lib/views/mentor/mentorscreen.dart (hunk 1 above)
Comment (ggpepp) [reply lib/views/mentor/mentorscreen.dart:33]: I dont understand what you mean

Comment (chwerth) [reply lib/views/mentor/mentorscreen.dart:33]: can you move the chat from being the resource screen to being the screen that pops up when you hit the chat button in the mentorprofilescreen? would that break anything?

Comment (IcemanOmikron) [reply lib/views/mentor/mentorscreen.dart:33]: So this one is the Mentor's view of the chat, not the Student's view of the chat, right? So it should stay as he has it.

**Review by IcemanOmikron — APPROVED** [review]
I haven't had the chance to run/test it, but am approving it in case you need the extra approval to merge it into master before the deadline.

File: lib/views/mentor/mentorscreen.dart (hunk 1 above)
Comment (chwerth) [reply lib/views/mentor/mentorscreen.dart:33]: My mistake - I meant the student's view of the chat. that one could be changed back now.

Regardless, the test would need to be fixed to look for the chat window in the resources tab in the mentor view

**Review by chwerth — APPROVED** [review]

**Pull request merged**